translations/pt/get5.phrases.txt

```diff
@@ -248,4 +356,84 @@
     {
         "pt"            "[NOT READY]"
```

Comment:
PRONT and NÃO PRONTO perhaps? Or would you not say that?

Reply:
Ready is a well-known term, I wouldn't change it

Reply:
Also NÃO PRONTO is strange to say